src/js/tabs/landing.js

```diff
@@ -2,19 +2,60 @@
 
 TABS.landing = {};
 TABS.landing.initialize = function (callback) {
-    var self = this;
+  var self = this;
 
-    if (GUI.active_tab != 'landing') {
-        GUI.active_tab = 'landing';
-    }
-
-    $('#content').load("./tabs/landing.html", function () {
-        // translate to user-selected language
-        i18n.localizePage();
+  if (GUI.active_tab != 'landing') {
+    GUI.active_tab = 'landing';
+  }
 
-        GUI.content_ready(callback);
+  $('#content').load("./tabs/landing.html", function () {
+    function showLang(newLang) {
+      var bottomSection = $('.languageSwitcher');
+      bottomSection.find('span').each(function(index) {
+        var element = $(this);
+        var languageSelected = element.attr('lang');
+        if (newLang == languageSelected) {
+          element.removeClass('selected_language');
+          element.addClass('selected_language');
+        } else {
+          element.removeClass('selected_language');
+        }
+      });
+    }
+    ConfigStorage.get('userLanguageSelect', function (result) {
```

Comment:
This was done like this in the "original" version, but now that we are modifying it, maybe is better to expose the method of `localization.js` that returns the language selected by the user. In this way the string `userLanguageSelect` will not need to be used outside the `localization.js` file. I don't know if the current implementation of this method can be used as is, but we can refactor it.